VitruvioHost/Plugins/Vitruvio/Source/VitruvioEditor/Private/VitruvioComponentDetails.cpp

```diff
@@ -135,7 +135,7 @@ void CreateColorPicker(const FLinearColor& InitialColor, C OnCommit)
 		PickerArgs.bOnlyRefreshOnOk = true;
 		PickerArgs.sRGBOverride = true;
 		PickerArgs.DisplayGamma = TAttribute<float>::Create(TAttribute<float>::FGetter::CreateUObject(GEngine, &UEngine::GetDisplayGamma));
-		PickerArgs.InitialColorOverride = InitialColor;
```

Comment:
Did they change API? On the official docs it's still InitialColorOverride:
https://docs.unrealengine.com/5.2/en-US/API/Runtime/AppFramework/Widgets/Colors/FColorPickerArgs/

Reply:
It's deprecated now.